src/Util/AutoloaderUtil.php

```diff
@@ -77,13 +72,23 @@ public function getNamespacePrefixForClass(string $className): string
     private function getClassLoader(): ClassLoader
     {
         if (null === self::$classLoader) {
-            $autoloadPath = $this->rootDir.'/vendor/autoload.php';
-
-            if (!file_exists($autoloadPath)) {
-                throw new \Exception(sprintf('Could not find the autoload file: "%s"', $autoloadPath));
+            $autoloadFunctions = spl_autoload_functions();
+            foreach ($autoloadFunctions as $autoloader) {
+                if (is_array($autoloader) && isset($autoloader[0]) && is_object($autoloader[0])) {
+                    if ($autoloader[0] instanceof ClassLoader) {
+                        self::$classLoader = $autoloader[0];
+                        break;
+                    } elseif (class_exists('Symfony\Component\Debug\DebugClassLoader')
```

Comment:
`class_exists` is useless. `$a instanceof NonExistentClass` is perfectly valid (it won't ever return `true` for non-existent classes) and this will avoid triggering the autoloader when it does not exist.

Reply:
Good to know. Changed it.